### DISCUSSION TITLE
Can someone update the doc with instructions for where to put the .neovintageous configuration file on Mac ?

### DISCUSSION THREAD

**Discussion by zaka634** [gerardroche/gerardroche.github.io#51] [General]

**Comment by gerardroche** [comment]
Use the Command Palette command: `NeoVintageous: Open RC File`

**Comment by gerardroche** [comment]
Don't forget to run the Command Palette command `NeoVintageous: Reload RC File` or restart sublime for any new mappings to take effect.

You can see more help via `:help nv`.

**Comment by gerardroche** [comment]
NeoVintageous creates the rc file in the User packages directory. You can find the user packages via the `Preferences > Browse Packages` menu. The rc file is created inside the `User` directory at `User/.neovintageousrc`
